src/components/Button.js

```diff
@@ -1,12 +1,30 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-function Button({ value, className }) {
-  return <button type="button" className={className}>{value}</button>;
+function Button({ className, value, onClick }) {
+  const handleButtonClick = () => {
+    onClick(value);
+  };
+
+  return (
+    <button type="button" className={className} onClick={handleButtonClick}>
+      {value}
+    </button>
+  );
 }
 
 export default Button;
+
 Button.propTypes = {
-  value: PropTypes.string.isRequired,
   className: PropTypes.string.isRequired,
+  value: PropTypes.string.isRequired,
+  onClick: PropTypes.func.isRequired,
 };
+
+// function Button({ className, value, onClick }) {
+//   return (
+//     <button type="button" className={className} onClick={() => onClick(value)}>
+//       {value}
+//     </button>
+//   );
+// }
```

Comment:
- [ ] Kindly remove all commented code from your project, you can leave comments about your code but not commented code, this will make your codebase look neat and readable 😉

Reply:
Thank you for your review @prolajumokeoni , I just removed the comments.